### ISSUE TITLE
Behaviour as the old versions of PHPCS, even after updating it

### ISSUE THREAD

**Issue by Pictor13** [SublimeLinter/SublimeLinter-phpcs#4]
I had a problem checking PHP code. 
Something like

```return Author::CLASS;```

is not recognised as expression (it expects a string, a variable, an opening curly brace, etc.).

Then I updated the 'phpcs' of my laptop, and if I manually sniff the code by shell, the same error is not reported.
But passing from Sublime Text I get the notification of wrong syntax.
I checked the SublimeLinter-phpcs settings but there is no particular path specified for 'phpcs', so it should use the same executable I was using in the shell (reachable though the PATH env-variable that is correctly set to see the executable).

How can I solve it?
How can I check the version of 'phpcs' that is used from SublimeText?

**Comment by ajmichels** [comment]
If you are still having this issue can you turn on SublimeLinter debugging, open the console, lint the file, and then paste the output here?

# Right Click on your file > SublimeLinter > Debug Mode
# View > Show Console
# Make a change in the file and copy the output in the console

**Comment by ajmichels** [comment]
Closing this since I haven't received a response from the submitter.
